frontend/sw360-portlet/src/main/webapp/js/main.js

```diff
@@ -249,3 +249,15 @@ function deleteConfirmed(confirmMessage, confirmCallback) {
     });
 }
 
+function stringToBoolean(s) {
+    return s == "true";
+}
+
+function hiddenIfNeccessary(PermissionsToDelete) {
```

Comment:
function parameter name beginning with an upper-case character - no bueno. Apart from that, if the function is in the main.js, it should look like it's rather generic. Like `styleAsHiddenIfNecessary(doHide)` or something like that.